lib/evm/operation/system.ex

```diff
@@ -73,136 +74,58 @@ defmodule EVM.Operation.System do
 
   ## Examples
 
-      # CALL
-      iex> block_interface = EVM.Interface.Mock.MockBlockInterface.new(%Block.Header{})
-      iex> account_map = %{<<0::160>> => %{balance: 5_000, nonce: 5}}
-      iex> account_interface = EVM.Interface.Mock.MockAccountInterface.new(%{account_map: account_map}, %{gas: 500, sub_state: nil, output: "output"})
-      iex> exec_env = %EVM.ExecEnv{stack_depth: 0, address: <<0::160>>, account_interface: account_interface, block_interface: block_interface}
-      iex> machine_state = %EVM.MachineState{gas: 300, stack: [1], memory: "________" <> "input"}
-      iex> %{machine_state: n_machine_state} =
-      ...>   EVM.Operation.System.call(
-      ...>     :call,
-      ...>     [100, <<0x01::160>>, 1_000, 5, 5, 1, 6],
-      ...>     %{exec_env: exec_env, machine_state: machine_state})
-      iex> n_machine_state
-      %EVM.MachineState{gas: 800, stack: [1, 1], active_words: 1, memory: "_output_input", program_counter: 0}
-
-      # CALLCODE
-      iex> block_interface = EVM.Interface.Mock.MockBlockInterface.new(%Block.Header{})
-      iex> account_map = %{<<0::160>> => %{balance: 5_000}}
-      iex> account_interface = EVM.Interface.Mock.MockAccountInterface.new(account_map, %{gas: 500, sub_state: nil, output: "output"})
-      iex> exec_env = %EVM.ExecEnv{stack_depth: 0, address: <<0::160>>, account_interface: account_interface, block_interface: block_interface}
-      iex> machine_state = %EVM.MachineState{gas: 300, stack: [1], memory: "________" <> "input"}
-      iex> %{machine_state: n_machine_state} =
-      ...>   EVM.Operation.System.call(
-      ...>     :call_code,
-      ...>     [100, <<0x01::160>>, 1_000, 5, 5, 1, 6],
-      ...>     %{exec_env: exec_env, machine_state: machine_state})
-      iex> n_machine_state
-      %EVM.MachineState{gas: 800, stack: [1, 1], active_words: 1, memory: "_output_input"}
-
-      # DELEGATECALL
-      iex> block_interface = EVM.Interface.Mock.MockBlockInterface.new(%Block.Header{})
-      iex> account_map = %{<<0::160>> => %{balance: 5_000}}
-      iex> account_interface = EVM.Interface.Mock.MockAccountInterface.new(account_map, %{gas: 500, sub_state: nil, output: "output"})
-      iex> exec_env = %EVM.ExecEnv{stack_depth: 0, address: <<0::160>>, account_interface: account_interface, block_interface: block_interface}
-      iex> machine_state = %EVM.MachineState{gas: 300, stack: [1], memory: "________" <> "input"}
-      iex> %{machine_state: n_machine_state} =
-      ...>   EVM.Operation.System.call(
-      ...>     :delegate_call,
-      ...>     [100, <<0x01::160>>, 5, 5, 1, 6],
-      ...>     %{exec_env: exec_env, machine_state: machine_state})
-      iex> n_machine_state
-      %EVM.MachineState{gas: 800, stack: [1, 1], active_words: 1, memory: "_output_input"}
-  """
-  @spec call(atom(), Operation.stack_args, Operation.vm_map) :: Operation.op_result
-  def call(:delegate_call, [call_gas, to, in_offset, in_size, out_offset, out_size], vm_map) do
```

Comment:
Just to verify, we're removing `DELEGATE_CALL` and `CALL_CODE` support for now?

Reply:
Yep, these methods wern't actually callable by the `VM`.

The VM calls `delegate_call` and `call_code` directly, instead of `call(:delegate_call...` and `call(:call_code...`

Hopefully I'll be adding them back soon...